tests/serial/map-if-necessary/two-solvers/mixed-substeps/SerialImplicit.cpp

```diff
@@ -0,0 +1,34 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(TwoSolvers)
+BOOST_AUTO_TEST_SUITE(MixedSubsteps)
+
+BOOST_AUTO_TEST_CASE(SerialImplicit)
+{
+  PRECICE_TEST("One"_on(1_rank), "Two"_on(1_rank));
+
+  // 1: iterating: new end
+  // 2: converged: new start and end
+  std::vector<int> readMappings{1, 2, 1, 2, 1, 1};
```

Comment:
```suggestion
  // 1: iterating: new end
  // 2: converged: new start and end
  std::vector<int> readMappings{1, 1, 1, 1, 1, 0};
```
but shouldn't the "new start" be the "end" of the previous time window? Hence the data was already mapped before? What am I not getting?

Reply:
Hmm. Right. The same holds for the other SerialImplicit cases.